### PR TITLE
Added email templates

### PR DESCRIPTION
**Pull request by bogdanprodan-okta**

### CODE REVIEW

File: dist/spec.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
       "name": "Apache-2.0",
       "url": "https://www.apache.org/licenses/LICENSE-2.0.html"
     },
-    "version": "2.10.0"
+    "version": "2.11.0"
```
Comment (monde) [inline dist/spec.json:16]: The version will be 2.12.0 as I released 2.11.0 for the IDP policy rules models.

Comment (monde) [reply dist/spec.json]: Never mind, this will likely be in 2.11.0

**Review by arvindkrishnakumar-okta — APPROVED** [review]
lgtm

**Review by monde — APPROVED** [review]
This works in the v2.11.0 RC branch for okta-sdk-golang https://github.com/okta/okta-sdk-golang/tree/release_2_11_0

**Comment by arvindkrishnakumar-okta** [comment]
closed in favor of #117 cc: @bretterer

**Pull request closed without merging**